core/iwasm/aot/aot_loader.c

```diff
@@ -2119,32 +2130,14 @@ load_globals(const uint8 **p_buf, const uint8 *buf_end, AOTModule *module,
 
     /* Create each global */
     for (i = 0; i < module->global_count; i++) {
-#if WASM_ENABLE_GC == 0
-        uint16 init_expr_type;
-#endif
-
         read_uint8(buf, buf_end, globals[i].type);
         read_uint8(buf, buf_end, globals[i].is_mutable);
+        /* Skip 2 byte for padding */
+        buf += 2;
```

Comment:
Is it better to use `buf = align_ptr(buf, 4);`, and ensure the same alignment when emitting AOT file?